src/fu-engine.h

```diff
@@ -26,13 +26,18 @@ G_DECLARE_FINAL_TYPE (FuEngine, fu_engine, FU, ENGINE, GObject)
  * FuEngineLoadFlags:
  * @FU_ENGINE_LOAD_FLAG_NONE:		No flags set
  * @FU_ENGINE_LOAD_FLAG_READONLY_FS:	Ignore readonly filesystem errors
+ * @FU_ENGINE_LOAD_FLAG_NO_ENUMERATE:	Do not enumerate devices
+ * @FU_ENGINE_LOAD_FLAG_NO_REMOTES:	Do not enumerate remotes
+ * @FU_ENGINE_LOAD_FLAG_NO_HWINFO:	Do not load details about the hardware
```

Comment:
These feel like they should also be fwupdtool arguments.

Reply:
I fixed this in a different way in https://github.com/fwupd/fwupd/pull/2668